plugins/quantum_k_means/tasks.py

```diff
@@ -38,6 +40,32 @@
 TASK_LOGGER = get_task_logger(__name__)
 
 
+def get_point(ent):
+    dimension_keys = list(ent.keys())
+    dimension_keys.remove("ID")
+    dimension_keys.remove("href")
+
+    dimension_keys.sort()
+    point = np.empty((len(dimension_keys, )))
+    for idx, d in enumerate(dimension_keys):
+        point[idx] = ent[d]
+    return point
+
+
+def get_entity_generator(entity_points_url: str, stream=False):
+    """
+    Return a generator for the entity points, given an url to them.
+    :param entity_points_url: url to the entity points
+    """
+    file_ = open_url(entity_points_url, stream=stream)
+    file_.encoding = "utf-8"
+    file_type = file_.headers["Content-Type"]
+    entities_generator = load_entities(file_, mimetype=file_type)
+    entities_generator = ensure_dict(entities_generator)
+    for ent in entities_generator:
+        yield {"ID": ent["ID"], "href": ent["href"], "point": get_point(ent)}
```

Comment:
entities may not have an href attribute! (then `ent["href"]` will throw an error)